### PR TITLE
Fix TreeView sorting after search (T563057)

### PR DESCRIPTION
**Pull request by DokaRus**

### CODE REVIEW

File: testing/tests/DevExpress.ui.widgets/treeViewParts/optionChanged.js [hunk 1]
```diff
@@ -88,6 +88,76 @@ QUnit.test("searchValue from value to empty", function(assert) {
     assert.equal($items.length, 6, "6 items were rendered after filtration");
 });
 
+QUnit.test("search should consider dataSource simple sorting", function(assert) {
```
Comment (sometimesElen) [inline testing/tests/DevExpress.ui.widgets/treeViewParts/optionChanged.js]: Seems that this test duplicates the dataAdapter test. You can expand that test and check it with a searchValue

File: testing/tests/DevExpress.ui.widgets/treeViewParts/optionChanged.js [hunk 2]
```diff
@@ -88,6 +88,76 @@ QUnit.test("searchValue from value to empty", function(assert) {
...
+
+    var $items = $(treeView.$element()).find(".dx-treeview-item"),
+        expectedValues = ["Cars", "Audi", "A1", "BMW", "X1", "Motobikes", "Yamaha", "YX 1"];
+
+    $.each($items, function(index, item) {
+        assert.equal($(item).text(), expectedValues[index], "Correct item");
+    });
+});
+
+QUnit.test("search should consider dataSource sorting expression", function(assert) {
```
Comment (sometimesElen) [inline testing/tests/DevExpress.ui.widgets/treeViewParts/optionChanged.js]: May be this test is extra, because it tests the store method?...

**Review by sometimesElen — CHANGES_REQUESTED** [review]
It will be enough in treeView to check if the sorting is added to dataAdapter on not.

File: testing/tests/DevExpress.ui.widgets/treeViewParts/optionChanged.js (hunk 2 above)
Comment (DokaRus) [reply testing/tests/DevExpress.ui.widgets/treeViewParts/optionChanged.js]: Nope. It tests that the data adapter receive a sorting expression and correctly handle it. I think it necessary to test this case, but probably better move it to the data adapter tests.

Comment (sometimesElen) [reply testing/tests/DevExpress.ui.widgets/treeViewParts/optionChanged.js]: Okey, sounds greater)

**Review by sometimesElen — APPROVED** [review]

**Pull request merged**